Example django-health-check (#302)
nautobot/nautobot: nautobot/docs/administration/nautobot-server.md

e!j=vrlhz-!wl8p_3+q5s5cph29nzj$xm81eap-!&n!_9^du09
```

### `health_check`

> `nautobot-server  health_check`

Reviewer: ```suggestion
`nautobot-server health_check`
```

> 

Reviewer: ```suggestion

Run health checks and exit 0 if everything went well.

```

```no-highlight

> nautobot-server health_check

Reviewer: ```suggestion
$ nautobot-server health_check
```

Cache backend: default   ... working
DatabaseBackend          ... working
DefaultFileStorageHealthCheck ... working
```

> Performs a health check of the server.  Please see the [healthcheck documentation](/additional-features/healthcheck/) for more information.

Reviewer: ```suggestion
Please see the [healthcheck documentation](../additional-features/healthcheck.md) for more information.
```
